src/game/engine/house.h

```diff
@@ -281,6 +283,11 @@ class HouseClass
     BOOL Hook_Is_Hack_Prevented(RTTIType rtti, int id) { return Is_Hack_Prevented(rtti, id); }
 #endif
 
+    fixed_t Get_Groundspeed_Multiplier() const { return m_GroundspeedMult; }
```

Comment:
These are better returned as `const fixed_t &` to avoid extra copy ctors being invoked.